app/models/service_ansible_playbook.rb

```diff
@@ -47,6 +47,16 @@ def postprocess(action)
 
   private
 
+  def manageiq_extra_vars
+    {
+      'MIQ_URL'          => MiqRegion.my_region.remote_ws_url,
+      'MIQ_TOKEN'        => Api::UserTokenService.new.generate_token(evm_owner.userid, 'api'),
+      'MIQ_SERVICE'      => id,
```

Comment:
I suggested using `id` until we get url slug support in PR https://github.com/ManageIQ/manageiq/pull/14344.  I would hold off merging this until then, but it gives us something to work with.